src/components/Header.spec.js

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import Header from './Header.js';
+
+describe('Header', () => {
+  // const cases = { linky: 'link === "y"', linkn: 'link !== "y"' };
```

Comment:
You can delete this.

Reply:
oh yeah, you are right, totally oversaw it🙈